package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "5.1.0",
```

Comment:
The @patternfly group needs to be bumped in non-minor version updates all together. 

In other words you need to bump all 4 depedencies.

Reply:
I cant change a version of a react-log-viewer because most recent version is 5.1.0-prerelease.1

Reply:
you can try if the 'mix' of versions works or wait. Up to you.